libs/qCC_db/ccPointCloudLOD.cpp

```diff
@@ -27,6 +27,8 @@
 //! Thread for background computation
 class ccPointCloudLODThread : public QThread
 {
+	Q_OBJECT
```

Comment:
Do you really need this when there's no signal/slot defined? I avoid putting it on purpose so as to avoid the creation and include of the MOC file.